persistence/assetlist.json

```diff
@@ -645,6 +645,37 @@
         "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/chihuahua/images/huahua.png",
         "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/chihuahua/images/huahua.svg"
       }
+    },
+    {
+      "description": "PSTAKE Liquid-Staked XPRT",
```

Comment:
For liquid staking tokens, please define the `traces`

Reply:
@JeremyParish69 done please check